packages/frontend/src/components/navbar/PageLink.tsx

```diff
@@ -9,15 +9,16 @@ export interface PageLinkProps {
 }
 
 export function PageLink(props: PageLinkProps) {
-  const className = classNames(
-    props.large ? 'text-base md:text-lg px-2 md:px-4' : 'px-2',
-    'flex items-center h-full',
-    props.selected &&
-      'pt-0.5 text-pink-900 dark:text-pink-100 border-current border-b-2',
-  )
-
   return (
-    <a className={className} href={props.href}>
+    <a
+      className={classNames(
```

Comment:
Nit: we sometimes import `classNames` as `cx`. Would probably make sense to unify this in the future.